File: src/OrchardCore.Modules/OrchardCore.Flows/Assets/ts/flow-part.ts
import observeAndInit from "@orchardcore/bloom/helpers/observeAndInit";
import initFlowSortablePlaceholder from "./flow-sortable-placeholder";

observeAndInit(".flowpart-sortable-placeholder", (placeholder) =>
    initFlowSortablePlaceholder(placeholder, "flowpart-widgets"),
);

File: src/OrchardCore/OrchardCore.Flows.Core/Models/FlowPart.cs
using System.ComponentModel;
using Microsoft.AspNetCore.Mvc.ModelBinding;
using OrchardCore.ContentManagement;

namespace OrchardCore.Flows.Models;

public class FlowPart : ContentPart
{
    [BindNever]
    [Description("The ordered widget content items in the flow.")]
    public List<ContentItem> Widgets { get; set; } = [];
}
